toolkit/tools/pkg/imagecustomizerlib/dracututils.go

```diff
@@ -63,66 +48,3 @@ func addDracutDriver(dracutDriverName string, imageChroot *safechroot.Chroot) er
 	}
 	return addDracutConfig(dracutConfigFile, lines)
 }
-
-func getDracutVersion(rootfsSourceDir string) (dracutPackageInfo *DracutPackageInformation, err error) {
```

Comment:
Removed since we can use the now-generalized `getPackageInformation()` (can handle versions with different component numbers (y, y.y, y.y.y, etc)).